packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -59,9 +68,41 @@ class CopyVmsModalBody extends BaseComponent {
   _onChangeNamePattern = event =>
     this.setState({ namePattern: event.target.value })
 
+  _getVmsWithNoZstdCompression = createSelector(
+    () => this.props.vms,
+    () => this.props.containers,
+    (vms, containers) => {
+      const vmIds = []
+      for (const id in vms) {
+        const container = containers[vms[id].$container]
+        if (!container.zstdSupported) {
+          vmIds.push(id)
+        }
+      }
+      return vmIds
```

Comment:
If the containers and/or the VMs change a lot, you could avoid a lot of renders just by adding a `createCollectionWrapper`.

Reply:
According to the [documentation](https://reactjs.org/docs/react-component.html#updating), if the props change React will rerender anyway. 

But if we use it here, will make sure that selectors which use it will return their cache if it not change.

Reply:
Yes but this would make the props unchanged for `Link` for example.